src/components/videoplayer/VideoPlayer.jsx

```diff
@@ -54,6 +59,8 @@ export const VideoPlayer = () => {
                 </div>
 
                 <p className='video-description'>{isVideoCardExist.description}</p>
+
+                <button  onClick={()=>likeVideo(videoListId, user.encodedToken)}>like</button>
```

Comment:
You should always use "user?.encodedToken" it will help to reduce error if encodedToken is undefined. It will not throw error and will not break the UI.